python/python/bystro/prs/tests/test_preprocess_for_prs.py

```diff
@@ -44,8 +48,8 @@ def mock_processed_scores_df() -> pd.DataFrame:
         "P": [0.699009, 0.0030673],
         "SNPID": ["chr1:566875:C:T", "chr1:728951:A:G"],
         "BETA": [0.007630, -0.020671],
-        "ID_effect_as_alt": ["chr1:566875:C:T", "chr1:728951:G:A"],
-        "ID_effect_as_ref": ["chr1:566875:T:C", "chr1:728951:A:G"],
+        "ID_effect_as_alt": ["chr8:132782505:C:T", "chr3:183978846:G:A"],
+        "ID_effect_as_ref": ["chr2:4000400:T:C", "chr21:24791946:C:T"],
```

Comment:
These two are supposed to have identical chrom and pos to SNPID and match each other